### PR TITLE
Add Spanish definition for "Secure Shell"

### PR DESCRIPTION
**Pull request by elsikorski**
## Author: 

- Ember Sikorski 

## Language: 

- Spanish

## Terms defined:

- Secure Shell

### CODE REVIEW

**Comment by nicoguaro** [comment]
@elsikorski, can you check my comments on this?

**Comment by elsikorski** [comment]
@nicoguaro thank you for reviewing, but I don't see any other comments here. I am new to pull requests - are they located somewhere else?

**Comment by nicoguaro** [comment]
@elsikorski, check [this](https://github.com/carpentries/glosario/pull/283/files/17556f72bf486d132bd8a8a8cccf471141bff0b9#diff-f13138bcde99805cc217314b608006094ad9d22f63e79a2f3cf019acc4a07003).

**Comment by jennyfothergill** [comment]
@nicoguaro No comments are visible. If your comment is part of a review, it won't show up until you hit "Submit Review."
Screen Shot 2021-04-12 at 11 27 13 AM

**Comment by nicoguaro** [comment]
> @nicoguaro No comments are visible. If your comment is part of a review, it won't show up until you hit "Submit Review."
> Screen Shot 2021-04-12 at 11 27 13 AM

Thank you, and sorry about that!

File: glossary.yml [hunk 1]
```diff
@@ -5574,6 +5574,13 @@
     acronym: "SSH"
     def: >
       A program that allows secure access to remote computers.
+  es:
```
[the remark this answers is not in the document]
Comment (elsikorski) [reply glossary.yml:5578]: I fixed a typo in the Spanish definition and modified the English one to match better.

**Review by nicoguaro — APPROVED** [review]

**Pull request merged**